ferveo/benches/benchmarks/eval_domain.rs

```diff
@@ -0,0 +1,57 @@
+#![allow(clippy::redundant_closure)]
+#![allow(clippy::unit_arg)]
+
+pub use ark_bls12_381::Bls12_381 as EllipticCurve;
+use ark_ff::Field;
+use ark_poly::EvaluationDomain;
+use criterion::{black_box, criterion_group, BenchmarkId, Criterion};
+use digest::crypto_common::rand_core::SeedableRng;
+use ferveo_pre_release::*;
+use rand::prelude::StdRng;
+
+const NUM_SHARES_CASES: [usize; 6] = [2, 4, 8, 16, 32, 64];
+
+pub fn bench_eval_domain(c: &mut Criterion) {
+    let mut group = c.benchmark_group("EVAL DOMAIN");
+    group.sample_size(10);
+
+    let rng = &mut StdRng::seed_from_u64(0);
+    let s = ark_bls12_381::Fr::from_random_bytes(&[0u8; 32]).unwrap();
+
+    for shares_num in NUM_SHARES_CASES {
```

Comment:
So this is benchmarking the speed of num shares that are powers of 2 for Radix2 vs MixedRadix? Cool.

As an aside. Do we have a test somewhere for a non-power of 2 number of shares? Basically, a test that confirms that the non-powers of 2 work appropriately.

Reply:
I didn't consider this kind of test to be interesting before. I did a manual test (in REPL) before submitting the PR and that lulled me into a false sense of correctness. Now that I followed up on your suggestion, it turns out only the simple variant works for a non-power-of-two number of shares. 

I will post the changes here and investigate the issue with the precomputed variant.

Thanks for catching this.

Reply:
The bug has been found and fixed. Please see my comment below.